HISTORY.md

```diff
@@ -1,6 +1,9 @@
+<!-- vale off -->
```

Comment:
I don’t understand how vale is off here, but there are still changes in the document

Reply:
Would it make sense to limit it to certain folders?

Reply:
Let’s start in this PR with `/docs` only, and then we can discuss if we want to extend it later.